bleep-cli/src/scala/bleep/Main.scala

```diff
@@ -502,28 +502,38 @@ object Main {
   }
 
   def run(logger: Logger, opts: Opts[BleepBuildCommand], started: Started, restArgs: List[String]): ExitCode =
-    Command("bleep", s"Bleeping fast build! (version ${model.BleepVersion.current.value})")(opts).parse(restArgs, sys.env) match {
-      case Left(help) =>
-        System.err.println(help)
-        ExitCode.Failure
-      case Right(cmd) =>
-        Try(cmd.run(started)) match {
-          case Failure(th)       => fatal("command failed", logger, th)
-          case Success(Left(th)) => fatal("command failed", logger, th)
-          case Success(Right(_)) => ExitCode.Success
-        }
-    }
+    Command("bleep", s"Bleeping fast build! (version ${model.BleepVersion.current.value})")(opts)
+      .parse(restArgs, sys.env)
+      .fold(
+        handleHelp,
+        cmd =>
+          Try(cmd.run(started)) match {
+            case Failure(th)       => fatal("command failed", logger, th)
+            case Success(Left(th)) => fatal("command failed", logger, th)
+            case Success(Right(_)) => ExitCode.Success
+          }
+      )
 
   def run(logger: Logger, opts: Opts[BleepNoBuildCommand], restArgs: List[String]): ExitCode =
-    Command("bleep", s"Bleeping fast build! (version ${model.BleepVersion.current.value})")(opts).parse(restArgs, sys.env) match {
-      case Left(help) =>
+    Command("bleep", s"Bleeping fast build! (version ${model.BleepVersion.current.value})")(opts)
+      .parse(restArgs, sys.env)
+      .fold(
+        handleHelp,
+        cmd =>
+          Try(cmd.run()) match {
+            case Failure(th)       => fatal("command failed", logger, th)
+            case Success(Left(th)) => fatal("command failed", logger, th)
+            case Success(Right(_)) => ExitCode.Success
+          }
+      )
+
+  private def handleHelp(help: Help): ExitCode =
+    help.errors match {
+      case Nil =>
+        System.err.println(help)
+        ExitCode.Success
+      case _ => // should we print the errors?
```

Comment:
I think `System.err.println(help)` takes care of it? You can try to println them to verify what is in there

Reply:
You're right, the toString implementation print errors if any